cfglib.py

```diff
@@ -418,7 +421,6 @@ def init_config(name_config=None):
         CONF.register_opts(i[1], i[0])
     if name_config:
         name_configs[0] = name_config
-    CONF(default_config_files=name_configs, args="")
```

Comment:
With `name_config=None` this line blanks all default option values. Needs to be fixed.